### PR TITLE
Add favicon

### PR DESCRIPTION
**Pull request by ebuildy**
Not a big one ;-)

Just copy your public site favicon.

### CODE REVIEW

File: client/app/index.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
     <link rel="icon" type="image/png" sizes="32x32" href="./assets/images/favicon-32x32.png">
     <link rel="icon" type="image/png" sizes="96x96" href="./assets/images/favicon-96x96.png">
     <link rel="icon" type="image/png" sizes="16x16" href="./assets/images/favicon-16x16.png">
+    <link rel="icon" href="./assets/images/favicon.ico">
```
Comment (arikfr) [inline client/app/index.html]: Isn't this  made redundant by the different sizes favicons above?

Comment (ebuildy) [inline client/app/index.html]: Ho actually, there is a 404 error:

https://demo.redash.io/assets/images/favicon-16x16.png

thats why the favicon is not showing :/

Comment (arikfr) [inline client/app/index.html]: The problem was wrong paths, but once they are sorted I don't think we need reference for "regular" favicon. See 3c7c93fc9f09cfa17319aef14da3d23ec6e46641.

**Comment by arikfr** [comment]
Closing. Please reopen if 3c7c93fc9f09cfa17319aef14da3d23ec6e46641 doesn't fix it for you.

Thanks!

**Pull request closed without merging**